src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/query-infrastructure-dimensional-metrics-nrql.mdx

```diff
@@ -66,7 +66,7 @@ For example, the query to get the maximum duration of your Lambda functions is s
 
 No agent or integration updates are required to use these metrics.
 
-[NRQL alerting](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries) based on dimensional metrics is also supported, except for data coming from [cloud integrations](/docs/integrations/infrastructure-integrations/cloud-integrations) (that is metrics from [AWS polling integrations](/docs/infrastructure/amazon-integrations/aws-integrations-list), [GCP](/docs/integrations/google-cloud-platform-integrations), and [Azure](/docs/integrations/microsoft-azure-integrations/azure-integrations-list)). AWS CloudWatch Metric Streams metrics are ingested as dimensional metrics and NRQL alerts are recommended. 
+[NRQL alerting](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries) based on dimensional metrics is also supported, except for data coming from cloud integrations (that is metrics from [AWS polling integrations](/docs/infrastructure/amazon-integrations/aws-integrations-list), [GCP](/docs/integrations/google-cloud-platform-integrations), and [Azure](/docs/integrations/microsoft-azure-integrations/azure-integrations-list)). AWS CloudWatch Metric Streams metrics are ingested as dimensional metrics and NRQL alerts are recommended. 
```

Comment:
I noticed that /docs/integrations/microsoft-azure-integrations/azure-integrations-list URL doesn't direct anywhere. So maybe we use the /docs/infrastructure/microsoft-azure-integrations/azure-integrations-list URL, which is redirecting to the IO site as expected. While you're at this,  maybe we edit all these URLs, because I see quite a few of that first type.

Reply:
Thanks! Where do I update the redirects to the IO site?

Reply:
Found it, and done. Tried locally and they work fine.